rogue-thi-app/pages/library.js

```diff
@@ -200,11 +202,11 @@ export default function Library () {
                     setReservationTime(time)
                   }}>{t('library.actions.reserve')}</Button>
 
-                  {formatNearDate(new Date(day.date + 'T' + time.from))}
+                  {formatNearDate(new Date(time.from))}
                   {', '}
-                  {formatFriendlyTime(new Date(day.date + 'T' + time.from))}
+                  {formatFriendlyTime(new Date(time.from))}
                   {' - '}
-                  {formatFriendlyTime(new Date(day.date + 'T' + time.to))}
+                  {formatFriendlyTime(new Date(time.to))}
```

Comment:
arent we turning these into Date objects in `library-utils.js` already?
afaik passing a Date to the Date constructor just creates a copy, so it should still work